src/target/adiv5.h

```diff
@@ -89,23 +89,19 @@
 #define ADIV5_AP_BASE		ADIV5_AP_REG(0xF8)
 #define ADIV5_AP_IDR		ADIV5_AP_REG(0xFC)
 
-/* Known designers seen in SYSROM-PIDR and JTAG IDCode.
- * Ignore Bit 0 from the designer bits to get JEDEC Ids.
- * Should get it's one file as not only related to Adiv5!
- */
+/* Known designers seen in SYSROM-PIDR. Ignore Bit 0 from
+ * the designer bits to get JEDEC Ids with bit 7 ignored.*/
 #define AP_DESIGNER_FREESCALE    0x00e
 #define AP_DESIGNER_TEXAS        0x017
 #define AP_DESIGNER_ATMEL        0x01f
 #define AP_DESIGNER_STM          0x020
 #define AP_DESIGNER_CYPRESS      0x034
 #define AP_DESIGNER_INFINEON     0x041
-#define DESIGNER_XILINX          0x049
 #define AP_DESIGNER_NORDIC       0x244
 #define AP_DESIGNER_ARM          0x43b
 /*LPC845 with designer 501. Strange!? */
 #define AP_DESIGNER_SPECULAR     0x501
 #define AP_DESIGNER_CS           0x555
-#define DESIGNER_XAMBALA         0x61e
```

Comment:
Same here as for the removed XILINX designer.

Reply:
Same reasoning as with the Xilinx code. We don't have any of their processors to test with though unlike Xilinx Zynq.